plugin/virustotal/vt_ida/disassembler.py

```diff
@@ -15,12 +15,119 @@
 
 import binascii
 import ida_idp
+import ida_search
+import ida_segment
 import idaapi
 import idautils
 import idc
 import logging
 
 
+class NavigateDisassembler(object):
+
+  @staticmethod
+  def go_to_ea(str_addr):
+    ea = int(str_addr, 0)
+    if ea is not None and ea is not idc.BADADDR:
+      idaapi.jumpto(ea)
+    else:
+      logging.debug('[VT Disassembler] Not valid EA: %s', str_addr)
+
+  @staticmethod
+  def is_head(ea):
+    flags = idc.get_full_flags(ea)
+    return idc.is_head(flags)
+
+  @staticmethod
+  def is_imports(segment):
+    if segment in ('.idata', '.rdata', '.edata'):
+      # Check: .dynstr, '.dynsym', __IMPORT
```

Comment:
Please go over all the code, and everywhere there is still work to be done, add

TODO(gerardofn): lo q sea